### PR TITLE
RatedUnqueue : BURST argument

### PR DESCRIPTION
**Pull request by tbarbette**
Allows to specify batch size in pull mode

### CODE REVIEW

**Comment by tbarbette** [comment]
I can't reproduce that segmentation fault locally... Weird. I'll give it more try.

File: elements/standard/ratedunqueue.cc [hunk 1]
```diff
@@ -55,6 +55,7 @@ RatedUnqueue::configure_helper(TokenBucket *tb, bool is_bandwidth, Element *elt,
     else
 	args.read_mp("RATE", r);
     if (args.read("BURST_DURATION", SecondsArg(3), dur_msec).read_status(dur_specified)
+	.read_or_set("BURST", ((RatedUnqueue*)elt)->_burst, 32)
```
Comment (piotrjurkiewicz) [inline elements/standard/ratedunqueue.cc]: Here. `RatedUnqueue::configure_helper` is called by elements, which are not subclasses of `RatedUnqueue`: `RatedSplitter` and `BandwidthRatedSplitter`.

**Comment by tbarbette** [comment]
Everything fixed now?

**Comment by piotrjurkiewicz** [comment]
It seems so.

**Pull request merged**